be1-go/message/test/messagedata/coin_post_transaction_test.go

```diff
@@ -36,7 +36,7 @@ func Test_Coin_Post_Transaction(t *testing.T) {
 	require.Equal(t, "P2PKH", msg.Transaction.Inputs[0].Script.Type)
 	require.Equal(t, "oKHk3AivbpNXk_SfFcHDaVHcCcY8IBfHE7auXJ7h4ms=", msg.Transaction.Inputs[0].Script.PubKey)
 	require.Equal(t, "6hY16bDKnb7bRA5j7IMDR1CJDqAJKOLuQRgKxdpYQIrtSTVTRjo5jigqPhYQEPcK5a1WAF86V739ENFnlp6YCw==", msg.Transaction.Inputs[0].Script.Sig)
-	require.Equal(t, 32, msg.Transaction.Outputs[0].Value)
+	require.EqualValues(t, 32, msg.Transaction.Outputs[0].Value)
```

Comment:
Why is this necessary, is there a limitation of Equal function?

Reply:
yes, Equal treats `uint64(32)` and `int(32)` as different. it's either that or changing the second argument

Reply:
Okay okay, agree then